docs/siem/siem-ui.asciidoc

```diff
@@ -1,102 +1,169 @@
+** Detection engine: Automatically searches for suspicious host and network
+activity via the following:
+*** {security-guide}/detection-engine-overview.html#detection-engine-overview[Detection rules]: Periodically search the data
+({es} indices) sent from your hosts for suspicious events. When a suspicious
+event is discovered, a detection alert is generated. External systems, such as
+Slack and email, can be used to send notifications when alerts are generated.
+You can create your own rules and make use of our {security-guide}/prebuilt-rules.html[prebuilt ones].
+*** {security-guide}/detections-ui-exceptions.html [Exceptions]: Reduce noise and the number of
+false positives. Exceptions are associated with rules and prevent alerts when
+an exception's conditions are met. *Value lists* contain source event
+values that can be used as part of an exception's conditions. When
+{es-sec-endpoint} is installed on your hosts, you can add malware exceptions
+directly to the endpoint from the Security app.
+*** {security-guide}/machine-learning.html#included-jobs[{ml-cap} jobs]: Automatic anomaly detection of host and
+network events. Anomaly scores are provided per host and can be used with
+detection rules.
+** {security-guide}/timelines-ui.html[Timeline]: Workspace for investigating alerts and events.
+Timelines use queries and filters to drill down into events related to
+a specific incident. Timeline templates are attached to rules and use predefined
+queries when alerts are investigated. Timelines can be saved and shared with
+others, as well as attached to Cases.
+** {security-guide}/cases-overview.html[Cases]: An internal system for opening, tracking, and sharing
+security issues directly in the Security app. Cases can be integrated with
+external ticketing systems.
+** {security-guide}/admin-page-ov.html[Administration]: View and manage hosts running {es-sec-endpoint}.
+
+{security-guide}/ingest-data.html[Ingest data to Elastic Security] and {security-guide}/install-endpoint.html[Configure and install the Elastic Endpoint integration] describe how to ship security-related
+data to {es}.
+
+
+For more background information, see:
+
+* https://www.elastic.co/products/elasticsearch[{es}]: A real-time,
+distributed storage, search, and analytics engine. {es} excels at indexing
+streams of semi-structured data, such as logs or metrics.
+* https://www.elastic.co/products/kibana[{kib}]: An open-source analytics and
+visualization platform designed to work with {es}. You use {kib} to search,
+view, and interact with data stored in {es} indices. You can easily perform
+advanced data analysis and visualize your data in a variety of charts, tables,
+and maps.
+
+[discrete]
+=== Compatibility with cold tier nodes
+
+Cold tier is a {ref}/data-tiers.html[data tier] that holds time series data that is accessed only occasionally. In {stack} version >=7.11.0, {es-sec} supports cold tier data for the following {es} indices:
```

Comment:
> In {stack} version >=7.11.0, {es-sec} supports ...

This is another attribute that doesn't seem to exist in https://github.com/elastic/docs/blob/master/shared/attributes.asciidoc at the moment. It either needs to be added there or a different existing attribute chosen from that file.